python/templates/DictLinkDef.h.jinja2

```diff
@@ -0,0 +1,27 @@
+{% macro class_selection(class, prefix='', postfix='') %}
+{%- if class.namespace %}
+        #pragma link C++ class {{ class.namespace }}::{{ class.bare_type }}{{ postfix }}+;
+{%- else %}
+        #pragma link C++ class {{ class.bare_type }}{{ postfix }}+;
+{%- endif %}
+{% endmacro %}
+// See: https://root.cern.ch/selecting-dictionary-entries-linkdefh
+#ifdef __CINT__
+
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+#pragma link C++ nestedclasses;
+
+{% for class in components %}
+{{ class_selection(class) }}
+{% endfor %}
+
+{% for class in datatypes %}
+{{ class_selection(class) }}
```

Comment:
Do we also need the `Mutable` ones here if we want to make them available in the python bindings? Or does this still work the same as with the `selection.xml` where things work (maybe due to the `Collection`)? For practical purposes this probably is of somewhat lower priority (see #431), but I would not like to accidentally break things.

Reply:
I think you are right we should make that explicit. Will include in the next update